src/blocks/CardChecklistBulletPointBlock.tsx

```diff
@@ -139,11 +146,11 @@ export const CardChecklistBulletPointBlock: React.FunctionComponent<
               />
               <div>
                 <BulletPointChecklist>
-                  {bullet.check_list.map((check) => {
+                  {bullet.check_list.map((check, id) => {
                     return (
-                      <BulletPointChecklistItem>
+                      <BulletPointChecklistItem key={id}>
```

Comment:
no index as key plz 🙈 can't we use `check._uid`?

Reply:
I'm getting lazy now towards the end :/

Reply:
haha :D